modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts

```diff
@@ -355,6 +355,8 @@ export type TxRequest = {
     signatureShares: SignatureShareRecord[];
     messageRaw: string;
     messageEncoded?: string;
+    messageBroadcastable?: string;
```

Comment:
i think we need to add commitmentShares here as well as we do in transactions. messages[idx].commitmentShares is being used in platform.

Reply:
I've added it, but it is not being used anywhere same as the `commitmentShares` of the `transactions` list.